app.py

```diff
@@ -72,9 +72,29 @@ def skill():
     Handles Skill requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        return jsonify([skill.__dict__ for skill in data["skill"]])
 
     if request.method == 'POST':
-        return jsonify({})
+        json_data = request.json
```

Comment:
I think the issue is about GET endpoint, right? It might be better to do the POST endpoint separately.

Reply:
Thanks. Noted and adjusted